app/models/request_form/requester_id.rb

```diff
@@ -2,14 +2,12 @@ module RequestForm
   class RequesterId < Base
     attribute :requester_photo
     attribute :requester_photo_id
-    attribute :requester_proof_of_address
-    attribute :requester_proof_of_address_id
+
     attr_accessor :default
 
     validates :requester_photo, presence: true, unless: -> { Attachment.exists?(requester_photo_id) }
     validates :requester_photo, file_size: { max: 7.megabytes }
-    validates :requester_proof_of_address, presence: true, unless: -> { Attachment.exists?(requester_proof_of_address_id) }
-    validates :requester_proof_of_address, file_size: { max: 7.megabytes }
+    validates :requester_photo, file_type: { allowed: %w[image/jpg image/jpeg image/png application/pdf application/doc application/docx] }
```

Comment:
Can you use the default options and combine with file_size like in requester_address?